taos-sys/build.rs

```diff
@@ -22,6 +22,9 @@ fn version2features<V: Into<Version>>(version: V) -> Vec<&'static str> {
         if version >= Version::new(3, 0, 2, 0) {
             feats.push("req_id");
         }
+        if version >= Version::new(3, 0, 4, 1) {
+            feats.push("tmq_offset_seek");
```

Comment:
Is it merged into main for version 3.0.4.2?

Reply:
It will merged for version 3.0.5.0